kolibri/plugins/management/assets/src/views/user-page/user-create-modal.vue

```diff
@@ -119,44 +117,61 @@
       LEARNER: () => UserKinds.LEARNER,
       COACH: () => UserKinds.COACH,
       ADMIN: () => UserKinds.ADMIN,
-      statusMessage() {
-        if (this.errorMessage) {
-          return this.errorMessage;
-        } else if (this.confirmationMessage) {
-          return this.confirmationMessage;
+      usernameAlreadyExists() {
+        const index = this.users.findIndex(user => user.username === this.username);
+        if (index === -1) {
```

Comment:
you could return `index !== 1` here and avoid the if statement. you could probably also use lodash `find` like:

`find(this.users, { username: this.username })`.